site/styles/_buttons.scss

```diff
@@ -129,6 +130,46 @@
   }
 }
 
+.s72-purchase-option.rent-or-buy {
+  .s72-purchase-option-buttons {
+    @include media-breakpoint-up(md) {
+      flex-wrap: nowrap;
+    }
+    // If there are two buttons...
+    :first-child:nth-last-child(2) {
```

Comment:
If I am reading this correctly the purpose of these styles is to add a 5px padding between the buttons if there are multiple buttons?

Could an alternative be:

```css
.s72-purchase-option-buttons {
    @include media-breakpoint-up(md) {
      flex-wrap: nowrap;

      .s72-btn-purchase-option { /* or whatever */
        margin-right: 5px;

       &:last-child: {
         margin-right: auto;
       }
    }

    ...
```

Alternatively you could move these styles to the `.s72-btn-purchase-option` block (see line 152+):

```css
.s72-btn-purchase-option {
  ...

  @include media-breakpoint-up(md) {
    margin-bottom: 0;
    margin-right: 5px; /* NEW */
  }

  &:last-child {
    margin-bottom: 0;
    margin-right: 0; /* NEW */
  }
}
```

The way this technique works is by adding margin to all things in the container then removing it from the last item. If there is only 1 then it will be the last, so the margin will be removed.